### ISSUE TITLE
.NET version v4.7.2 for Windows 10, 1803

### ISSUE THREAD

**Issue by jmack-87** [EDCD/EDDI#582]
## EDDI version in which issue found
3.0.1-b2, VA v1.7.0.6
Windows 10, 1803 (.NET v4.7.2)

## Steps to reproduce

1. install VA v1.7.0.6
2. install EDDI 3.0.1-b2
3. configure EDDI as usual
4. launch VA
5. observe very long load time or silent crash of VA+EDDI plugin

## Expected
VA launches and EDDI plugin initializes nearly instantaneously (on SSD)

## Observed
VA launches, but plugin loading is delayed ~60s.
-OR-
VA launches, plugin loading never occurs. VA crashes (silent)

## Investigation
modified EddiVoiceAttackResponder.dll.config
```xml
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2"/></startup></configuration>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.1"/></startup></configuration>
```

expected behavior is restored using `v4.7.2`.

[VoiceAttackFault.txt](https://github.com/EDCD/EDDI/files/2029075/VoiceAttackFault.txt)
[eddi-truncated.log](https://github.com/EDCD/EDDI/files/2029086/eddi-truncated.log)

**Comment by Tkael** [comment]
Interesting. I think it is likely a duplicate of #539 and if so we have a different fix already prepped for the next release. 
Why did you choose to advance EddiVoiceAttackResponder.dll.config to .Net 4.7.2 to correct this issue?

**Comment by jmack-87** [comment]
TL;DR: Lucky shot in the dark.

Looking at the VoiceAttackFault logs, I noticed several SQLite database locked errors.
A google search for "voice attach sqlite database locked" returned the following [stackoverflow](https://stackoverflow.com/questions/22384460/receiving-database-is-locked-error-with-sqlite-and-c-sharp?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa).
The single answer mentioned .NET.

Since the issue cropped up with the Win10 update to 1803, I checked into any .NET changes delivered with 1803. [Microsoft](https://docs.microsoft.com/en-us/dotnet/framework/install/on-windows-10) listed `v4.7.2` as the most current .NET package and the same delivered with Windows 10 April 2018 update (1803). I had hoped for a disparity on my end solvable with a .NET update. No dice.

Research was getting me nowhere until I considered that it was the plugin with the issue, not EDDI proper. So, I went digging for plugin logs (if any existed). During my search, I encountered **EddiVoiceAttackResponder.dll.config**. I looked inside.

Therein I found the reference to `v4.7.1`. I figured, 'Worth a shot.' Change it to `v4.7.2`, launched VoiceAttack (configured for EDDI plugin and profile on launch), counted to 5s, and _BAM_, that beautiful green dot appeared :-)
![image](https://user-images.githubusercontent.com/32743817/40403501-50ee245c-5e20-11e8-8e8d-f0d393ffd913.png)

Sorry the story wasn't more technical.

**Comment by Tkael** [comment]
It very well might have been a coincidence. :-)
Our conclusion was that we had a thread that was going a bit amuck (throwing out all of those unknown commodity messages) and that this was locking access to the sqlite database. By addressing those issues we now have a build that is much more stable.
I'm going to close this issue for the time being, but I'm glad you found a solution. We're moving towards a new beta release soon.
